src/mlir/generator.h

```diff
@@ -46,6 +46,10 @@ namespace mlir::verona
     /// Default constructor, builds an empty return value.
     ReturnValue() {}
     /// Single value constructor.
```

Comment:
Comment moved function body? Is that correct, should both functions have the same comment.

Reply:
Actually, that was meant to be a replacement. `mlir::Value` is meant to always be passed by value. Removed the reference version.

But yes, the comment is still relevant.